t/ui/14-dashboard.t

```diff
@@ -27,7 +27,7 @@ use OpenQA::SeleniumTest;
 
 OpenQA::Test::Case->new->init_data(fixtures_glob => '01-jobs.pl 02-workers.pl 03-users.pl');
 
-my $driver = call_driver();
+my $driver = call_driver;
 unless ($driver) {
     plan skip_all => $OpenQA::SeleniumTest::drivermissing;
```

Comment:
Might even go one step further and use a one-liner like in `t/ui/13-admin-no-login.t` everywhere

Reply:
I will. Expect different follow-up PR